File: BookHotel/BookHotel/Data/Entities/City.cs
using System.ComponentModel.DataAnnotations;
using System.Globalization;
using BookHotel.Auth.Model;

namespace BookHotel.Data.Entities
{
	public class City : IUserOwnedResource
	{
		public int Id { get; set; }
		public string CityName { get; set; }
		public string ImageUrl { get; set; }
		public string County { get; set; }
		[Required]
		public string UserId { get; set; }
		public BookHotelRestUser User { get; set; }

	}
}

File: bookhotel-frontend/src/cities/models/City.ts
export default class City {
  id: string;
  county: string;
  cityName: string;
  imageUrl: string;

  constructor(dto: any) {
    this.id = dto._id;
    this.cityName = dto.cityName;
    this.county = dto.county;
    this.imageUrl = dto.imageUrl ?? '';
  }
}
